src/Utils/Utils.php

```diff
@@ -261,4 +261,28 @@ public static function get_node_type_from_id( $id ) {
 		$id_parts = Relay::fromGlobalId( $id );
 		return $id_parts['type'] ?: null;
 	}
+
+	/**
+	 * Given a WP Post or post ID, this method attempts to resolve a preview post ID.
+	 * 
+	 * @param Int|\WPGraphQL\Utils\WP_Post $post The WP Post object or Post ID
+	 * 
+	 * @return Int A preview post ID if one exists, the current post ID if one doesn't exist.
+	 */
+	public static function get_post_preview_id( $post ) {
```

Comment:
It's a new method, lets be strict here.
```suggestion
	 */
	public static function get_post_preview_id( $post ): int {
```

Reply:
Whoops. accidentally removed the entire return comment. Updated suggestion.